### PR TITLE
Action row aligning fixes, color token changes, and icon fixes

### PR DESCRIPTION
**Pull request by arrenv**
## Description

This PR makes a lot of very small bug fixes, as well as some UI/UX improvements, including:
- Alignment of action rows.
- Corrected order of rows.
- Spacing and padding of action rows.
- Corrected icon sizes.
- Corrected token avatar size.
- Alignment of multiline content.
- Improvements to description field.
- Removal of redundant and intrusive tooltips.
- Removing misleading cursors.
- Removing default values that resulted in bad user experience.
- Fixes and updates to color tokens.
- Changes to font colors with direction from design.
- Removal of hover state on `MemberPopover` and member kebab menu.

I originally started breaking these up into individual branches, but it started to become harder to track, left me know if you would prefer this though and I can put a little more time into it.

**Before**
image

**After**
image

### CODE REVIEW

**Review by TomaszChmielPagepro — APPROVED** [review]

**Review by melyndav — APPROVED** [review]

File: src/components/v5/common/ActionSidebar/partials/AmountField/hooks.ts [hunk 1]
```diff
@@ -30,10 +30,11 @@ export const useAmountField = (selectedTokenAddress: string) => {
 
   const onInput = (event: React.ChangeEvent<HTMLInputElement>) => {
     const { value } = event.target;
-    const valueWithoutCommas = value.replace(/,/g, '');
-    const width = valueWithoutCommas.length * 0.65;
-
-    setInputWidth(width);
+    if (/^[\d,]*$/.test(value)) {
```
Comment (jakubcolony) [inline src/components/v5/common/ActionSidebar/partials/AmountField/hooks.ts:33]: Can that regex be stored in a meaningfully named variable or a comment be added to what it checks for? It's a bit hard to tell on first look

File: src/components/v5/common/ActionSidebar/partials/forms/CreateDecisionForm/consts.ts [hunk 2]
```diff
@@ -6,11 +6,11 @@ export const validationSchema = yup
     title: yup
       .string()
       .trim()
-      .required(() => 'Please enter a title'),
+      .required(() => 'Please enter a title.'),
     createdIn: yup.string().defined(),
     description: yup
       .string()
-      .notOneOf(['<p></p>'], () => 'Please enter a description')
+      .notOneOf(['<p></p>'], () => 'Please enter a description.')
```
Comment (jakubcolony) [inline src/components/v5/common/ActionSidebar/partials/forms/CreateDecisionForm/consts.ts:13]: Shouldn't those messages come from react-intl?

**Review by jakubcolony — APPROVED** [review]
LGTM

File: src/components/v5/common/ActionSidebar/partials/forms/CreateDecisionForm/consts.ts (hunk 2 above)
Comment (arrenv) [reply src/components/v5/common/ActionSidebar/partials/forms/CreateDecisionForm/consts.ts:13]: Yes, this is actually quite a big one, with other places I have noticed the same. It is probably best done in another PR.

**Pull request merged**